srcpkgs/btdu/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'btdu'
+pkgname=btdu
+version=0.3.0
+revision=1
+archs=x86_64
```

Comment:
Add a comment that 
a) druntime does not implement ioctl on musl in dmd-2.098.0
b) struct packing sems to differ on i686 archs and therefore d-btrfs fails